.github/workflows/openbsd.yaml

```diff
@@ -0,0 +1,66 @@
+name: Staged commit - OpenBSD
+
+on:
+  push:
+    branches: [ "bsd-merge-tests" ]
+
+env:
+  # empty except for pull_request events
+  PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
```

Comment:
I cannot find where PULL_REQUEST_NUMBER is used by this workflow. Please remove as unused or let me know what I have missed.